build.sh

```diff
@@ -74,9 +80,13 @@ function printHelp() {
     {
         printf " \t%s\t%s\n" "--test" "to run the tests"
         printf " \t%s\t%s\n" "--testUi" "to run the tests including UI tests"
+        printf " \t%s\t%s\n" "--installCore" "install JMC core"
         printf " \t%s\t%s\n" "--packageJmc" "to package JMC"
+        printf " \t%s\t%s\n" "--packageAgent" "to package Agent"
         printf " \t%s\t%s\n" "--clean" "to run maven clean"
         printf " \t%s\t%s\n" "--run" "to run JMC once it was packaged"
+        printf " \t%s\t%s\n" "--runAgentExample" "to run Agent Example once it was packaged"
+        printf " \t%s\t%s\n" "--runAgentCustomExample --class:'org.openjdk.jmc.agent.test.InstrumentMe'" "to run custom agent class, once Agent was packaged"
```

Comment:
This line is really long and causes multi-lines at small widths; it's difficult to read the menu. 

Does this line do anything at the moment? It doesn't have a match in `parseArgs`

Reply:
yeah it does, @aptmac I've update the repo now, maybe I'd remove the Example or just merge them together,--runAgent* is the terminal option, it returns the error code. what do you think ?